README.md

```diff
@@ -48,12 +62,33 @@ import SwiftCurrent
 launchInto(Workflow(ExampleViewController.self))
 ```
 
+### Original:
```

Comment:
There's always option 3, which is to not even call out the example app or the getting started.  And just end this section.

Reply:
This option was the winner